ra/ra_test.go

```diff
@@ -1407,6 +1407,19 @@ func TestCheckCertificatesPerNameLimit(t *testing.T) {
 	if !berrors.Is(err, berrors.RateLimit) {
 		t.Errorf("Incorrect error type %#v", err)
 	}
+	// Verify it has no sub errors as there is only one bad name
+	test.AssertEquals(t, len(err.(*berrors.BoulderError).SubErrors), 0)
+
+	// Two base domains, above threshold
```

Comment:
Can you update this test case to add a few more names that **aren't** out of rate limit to the `[]string` of domains given to `checkCertificatesPerNameLimit`? I think the rest of the test can remain as-is, the number of errors/format of them should remain the same and not include the names that aren't out of rate limit.